Meshconvert/Mesh.cpp

```diff
@@ -894,7 +894,8 @@ HRESULT Mesh::GetVertexBuffer(_Inout_ DirectX::VBWriter& writer) const
         auto e = writer.GetElement11("NORMAL", 0);
         if (e)
         {
-            hr = writer.Write(mNormals.get(), "NORMAL", 0, mnVerts);
+            bool x2bias = (e->Format == DXGI_FORMAT_R11G11B10_FLOAT);
```

Comment:
``DXGI_FORMAT_R11G11B10_FLOAT`` can be a great normal format, but it's unsigned only, so to encode normal vectors, you have to x2 bias it.